src/gc-debug.c

```diff
@@ -983,13 +983,13 @@ void gc_time_summary(int sweep_full, uint64_t start, uint64_t end,
                      uint64_t pause)
 {
     if (sweep_full > 0)
-        jl_safe_printf("%ld Major collection: estimate freed = %ld
-                       live = %ldm new interval = %ldm time = %ldms\n",
+        jl_safe_printf("%ld Major collection: estimate freed = %ld\n"
+                       "live = %ldm new interval = %ldm time = %ldms\n",
                        end - start, freed, live/1024/1024,
                        interval/1024/1024, pause/1000000 );
     else
-        jl_safe_printf("%ld Minor collection: estimate freed = %ld live = %ldm
-                       new interval = %ldm time = %ldms\n",
+        jl_safe_printf("%ld Minor collection: estimate freed = %ld live = %ldm\n"
+                       "new interval = %ldm time = %ldms\n",
```

Comment:
I'm not even sure how this compiled, but I presume this was the intention of these printfs?

Reply:
I am not sure it intended to have newlines

Reply:
@chflood how where these lines intended to be printed?  A standard `printf` wouldn't even compile with the previous syntax.  I'm happy to fix it according to the original intention.